batch.py

```diff
@@ -694,6 +694,20 @@ def crm_pipeline(
     celltype_list = celltypes.split(' ')
     logging.info(f'Cell types to run: {celltype_list}')
 
+    # only run this for relevant genes
+    plink_genes: List[str] = []
+    for celltype in celltype_list:
+        expression_tsv_path = dataset_path(
+            os.path.join(
+                expression_files_prefix,
+                'expression_files',
+                f'{celltype}_expression.tsv',
+            )
+        )
+        plink_genes.extend(extract_genes(genes_of_interest, expression_tsv_path))
+    plink_genes = list(sorted(set(plink_genes)))
+    print(f'Genes: {plink_genes}')
```

Comment:
Just generally speaking, does this print a lot of stuff? Print statement are good for debugging, unless they clog up the logs.